tensorflow/compiler/xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -47,6 +47,34 @@ using tensorflow::AutotuneResult;
 
 namespace {
 
+StatusOr<se::cuda::BlasLt::Epilogue> AsBlasLtEpilogue(
+    GemmBackendConfig_Epilogue epilogue) {
+  switch (epilogue) {
+    case GemmBackendConfig::DEFAULT:
+      return se::cuda::BlasLt::Epilogue::kDefault;
+    case GemmBackendConfig::RELU:
+      return se::cuda::BlasLt::Epilogue::kReLU;
+    case GemmBackendConfig::BIAS:
+      return se::cuda::BlasLt::Epilogue::kBias;
+    case GemmBackendConfig::BIASRELU:
+      return se::cuda::BlasLt::Epilogue::kBiasThenReLU;
+    case GemmBackendConfig::GELU:
```

Comment:
It's fine that we don't pattern-match GELU and BIASGELU in this patch, but please add comments indicating that this is incomplete.